joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/dataflow/DataflowTest.scala

```diff
@@ -586,4 +586,15 @@ class DataflowTest extends DataFlowCodeToCpgSuite {
     println(sink.reachableBy(src).l)
```

Comment:
While reviewing this, I noticed that this test here is also broken: could you possibly replace this with `sink.reachableBy(src).size shouldBe 1` as part of your PR? Many thanks!

Reply:
> While reviewing this, I noticed that this test here is also broken: could you possibly replace this with `sink.reachableBy(src).size shouldBe 1` as part of your PR? Many thanks!

Will try!